Generate Java data models with javax validation constraints annotations (#428)
asyncapi/modelina: docs/languages/Java.md

TODO

## Include javax validation constraints annotations for properties

> In some cases where you generate the models from JSON Schema and you might want them to include `javax.validation.constraint` annotations.
> 
> Check out this [example out for a live demonstration](../../examples/java-generate-javax-constraint-annotation).

Reviewer: ```suggestion
In some cases, when you generate the models from JSON Schema, you may want to include `javax.validation.constraint` annotations.

Check out this [example for a live demonstration](../../examples/java-generate-javax-constraint-annotation).
```

> ## Include Jackson annotations for the class

Reviewer: Jackson?

Reviewer 2: See this @alequetzalli 🙂 -> https://github.com/asyncapi/modelina/issues/427
